src/Microsoft.AspNetCore.Mvc.Razor/RazorViewEngine.cs

```diff
@@ -415,6 +416,14 @@ public string GetAbsolutePath(string executingFilePath, string pagePath)
                 var viewStartPages = isMainPage ?
                     GetViewStartPages(relativePath, expirationTokens) :
                     EmptyArray<ViewLocationCacheItem>.Instance;
+                if (factoryResult.IsPrecompiledView)
+                {
+                    _logger?.PrecompiledViewFound(relativePath);
+                }
+                else
+                {
+                    _logger?.PrecompiledViewNotFound(relativePath);
```

Comment:
This isn't particularly useful information. I'd leave the precompiled view found message and remove this one.